### DISCUSSION TITLE
Any idea if this could work on a console?

### DISCUSSION THREAD

**Discussion by NitroPlum** [gen2brain/raylib-go#323] [Q&A]
I doubt you have tried it, but do you know if this has any chance of running on a console? I know Ebiten can, but WASM is limited because of cgo so I wasn't sure if that might break that possibility as well.

**Comment by gen2brain** [comment]
Sure, it does support running on Linux `DRM` console, and input is handled via `evdev`, but it also supports plain keyboard input to run over `SSH`. But I am not sure what "console" you are talking about, how is it related to WASM?

**Reply by NitroPlum** [reply]
Sorry I wasn't clear enough. I mean a game console. Like would it be possible to run a game made in Golang + raylib on a Nintendo Switch for example.

**Reply by gen2brain** [reply]
After the platform split, an SDL backend can be used, so for some gaming consoles, it should be just a question of compiling with the appropriate toolchain. The creator of Raylib already ported it to Nintendo Switch, but due to license issues it cannot be open-sourced, also, it must be C++ if I am not mistaken, something like that. If I remember correctly, Ebiten used a transpiler from Go to C++ just for that.
